bin/build_project.sh

```diff
@@ -95,7 +95,16 @@ fi
 if [ "$AOMP_LEGACY_OPENMP" != 0 ]; then
   LLVM_RUNTIMES="libcxx;libcxxabi;libunwind;compiler-rt"
 else
-  LLVM_RUNTIMES="libcxx;libcxxabi;libunwind;openmp;offload;compiler-rt;flang-rt"
```

Comment:
flang project enables flang-rt , so it does not need to be in the LLVM_RUNTIMES,   this is recent change that Kruse can confirm.